elide-datastore/elide-datastore-aggregation/src/test/java/com/yahoo/elide/datastores/aggregation/AggregationDataStoreIntegrationTest.java

```diff
@@ -226,6 +439,273 @@ public void whereFilterTest() throws Exception {
+                                        field("overallRating", "Good"),
+                                        field("lowScore", 72),
+                                        field(
+                                                "player",
+                                                selections(
+                                                        field("name", "Han")
+                                                )
+                                        )
+                                ),
+                                selections(
+                                        field("overallRating", "Good"),
+                                        field("lowScore", 35),
+                                        field(
+                                                "player",
+                                                selections(
+                                                        field("name", "Jon Doe")
+                                                )
+                                        )
+                                ),
+                                selections(
+                                        field("overallRating", "Great"),
+                                        field("lowScore", 241),
+                                        field(
+                                                "player",
+                                                selections(
+                                                        field("name", "Jane Doe")
+                                                )
+                                        )
+                                )
+                        )
+                )
+        ).toResponse();
+
+        runQueryWithExpectedResult(graphQLRequest, expected);
+    }
+
+    @Test
+    public void invalidSortingTest() throws Exception {
+        String graphQLRequest = document(
+                selection(
+                        field(
+                                "playerStats",
+                                arguments(
+                                        argument("sort", "\"highScore\"")
+                                ),
+                                selections(
+                                        field("lowScore"),
+                                        field(
+                                                "country",
+                                                selections(
+                                                        field("name")
+                                                )
+                                        )
+                                )
+                        )
+                )
+        ).toQuery();
+
+        String expected = "\"Exception while fetching data (/playerStats) : Invalid operation: 'Can't sort on metric that is not present in query'\"";
+
+        runQueryWithExpectedError(graphQLRequest, expected);
+    }
+
+    @Test
+    public void invalidQueryTest() throws Exception {
```

Comment:
Let's make the function name more explicit about what is invalid here.  We'll probably have lots of invalid query tests.